modules/account-lib/src/coin/cspr/transactionBuilder.ts

```diff
@@ -217,6 +217,14 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
     if (this._fee === undefined) {
       throw new BuildTransactionError('Invalid transaction: missing fee');
     }
+    if (!this._fee.gasLimit) {
+      throw new BuildTransactionError('Invalid transaction: missing gas limit');
+    }
+    try {
+      this.validateValue(new BigNumber(this._fee.gasLimit, 16));
```

Comment:
is `this._fee.gasLimit` set in base 16?

Reply:
No, it's a mistake to set the base here.
I'll remove it. Thanks @CapnMigraine !